subprojects/core/src/integTest/groovy/org/gradle/api/ConfigurationOnDemandIntegrationTest.groovy

```diff
@@ -471,9 +471,8 @@ allprojects {
         fixture.assertProjectsConfigured(":", ":a", ":b")
     }
 
-    @Issue("https://github.com/gradle/gradle/issues/29154")
-    @ToBeFixedForIsolatedProjects(because = "-x is not IP compatible")
-    def "does not configure all projects when excluded task path is not qualified and an exact match for task has already been seen in some sub-project of default project"() {
+    @ToBeFixedForIsolatedProjects(because = "cross project access")
```

Comment:
🧪 I might be missing something because I see the tests have been updated but they are all still marked `@ToBeFixedForIsolatedProjects`. Which test proves this change has the intended effect?